src/linear_algebra/direct_sum_module.lean

```diff
@@ -35,7 +35,7 @@ namespace direct_sum
 
 variables {R ι M}
 
-instance : semimodule R (direct_sum ι M) := dfinsupp.to_semimodule
+instance : semimodule R (direct_sum ι M) := @dfinsupp.to_semimodule ι M _ _ (λ i, by apply_instance) _
```

Comment:
Without this lean can't show `[Π i, add_comm_monoid (β i)]` from `[Π i, add_comm_group (β i)]` it seems

Reply:
I think you want to import algebra.pi_instances